src/main/java/race/vehicle/Car.java

```diff
@@ -0,0 +1,46 @@
+package race.vehicle;
+
+import race.Utils.NumberGenerator;
+
+public class Car {
+    private static final int MOVE_CONDITION = 4;
+    private static final int BOUND = 10;
+    public static final String INITIAL_LOCATION = "-";
+
+    private StringBuffer location = new StringBuffer(INITIAL_LOCATION);
+    private NumberGenerator numberGenerator;
```

Comment:
```suggestion
    private final StringBuffer location = new StringBuffer(INITIAL_LOCATION);
    private final NumberGenerator numberGenerator;
```
로 불변하게 만들 수 있을 것 같네요 😄